memote/support/helpers.py

```diff
@@ -367,6 +367,31 @@ def find_exchange_rxns(model):
             if extracellular in rxn.get_compartments()]
 
 
+def find_tra_bou_bio_reactions(model, biomass=None):
```

Comment:
As I mentioned I'm really not a fan of abbreviated function and argument names. Internally, that's totally fine with me but externally not so much.

Reply:
Yeah, that was the name Christian suggested, so I just stuck with it. I can change it to something else